linetools/tests/test_init_absline.py

```diff
@@ -26,6 +27,12 @@ def test_mk_absline():
     abslin2 = AbsLine(1334.5323*u.AA, linelist='Strong', use_CACHE=True)
     np.testing.assert_allclose(abslin2.data['Ek'], 74932.62 / u.cm)
 
+    # Init CII 1334 with multiple linelists
+    ism = LineList('ISM')
+    H2 = LineList('H2')
+    abslin2b = AbsLine(1334.5323*u.AA, linelist=[ism,H2])
```

Comment:
can you invert the order of the linelist for the test? i.e. `[H2, ism]`

Reply:
done